servicemigration/context.py

```diff
@@ -41,7 +41,10 @@ def __init__(self, filename=None):
         else:
             infile = os.environ["MIGRATE_INPUTFILE"] if "MIGRATE_INPUTFILE" in os.environ else None
 
-        self._client = ControlPlaneClient(**getConnectionSettings())
+	if ZenUtils:
```

Comment:
This change was simply to enable running the tests.  It was the only change in the [Fix unit tests] (https://github.com/control-center/service-migration/pull/44/commits/e09edacfbfc09056ccb8f91cfeb04abe364ca2be) commit.  
The tests were broken a [few weeks ago](https://github.com/control-center/service-migration/pull/43); nobody noticed until now.